msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -207,13 +210,11 @@ class AccountState private constructor(
     interface GetAccessTokenCallback : Callback<GetAccessTokenResult>
 
     /**
-     * Retrieves the access token for the currently signed in account from the cache.
+     * Retrieves the access token for the default OIDC(openid, offline_access, profile)  scopes from the cache
      * If the access token is expired, it will be attempted to be refreshed using the refresh token that's stored in the cache;
      * callback variant.
      *
      * @return [com.microsoft.identity.client.IAuthenticationResult] If successful.
-     * @throws [MsalClientException] If the the account doesn't exist in the cache.
-     * @throws [ServiceException] If the refresh token doesn't exist in the cache/is expired, or the refreshing fails.
```

Comment:
Why are these `throws` removed?

Reply:
This method doesn't throw exception after PR #2080 . I updated the doc to be inline with the current functionality.